ecosystem/typescript/sdk/README.md

```diff
@@ -74,14 +79,20 @@ This project follows [semver](https://semver.org/) as closely as possible
 [api-doc]: https://aptos-labs.github.io/ts-sdk-doc/
 
 ## Release process
+To release a new version of the SDK do the following.
 
-1. Regenerate generated types `npx swagger-typescript-api@latest -p ../../../api/doc/v0/openapi.yaml -o ./src/api --modular --axios --single-http-client`
-2. Update your commit message to follow the Angular Conventional Commits
-3. Test lint and format `yarn test` `yarn lint` `yarn fmt`
-4. Update the version in the `package.json` file and run `yarn changelog` to generate the changelog
-5. Once it's all committed you can run npm release
+1. Regenerate the client:
+```
+yarn generate-client
+```
+
+2. Test, lint and format:
+```
+yarn test
+yarn lint
+yarn fmt
+```
 
-Generate the change log. yarn changelog and bump the version in package.json. Only bump the minor version.
-Commit the changes.
-yarn build to build the package. Ideally, you should test the new package in browser to make sure it works in browser as well.
-Follow https://aptos-org.slack.com/archives/C034HFWPJ05/p1651687417201819, to publish the package
+3. Ensure your commit message adheres to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification. TODO, provide an example of how to do this, this is not very well known.
```

Comment:
Create an issue for the TODO. We use too many TODOs and we rarely get back to those. Creating an issue is better.